contracts/mocks/ERC721AStartOneMock.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+// Creators: Chiru Labs
+
+pragma solidity ^0.8.4;
+
+import '../ERC721A.sol';
+
+contract ERC721AStartOneMock is ERC721A {
```

Comment:
You should just inherit from `ERC721AMock` instead of redeclaring everything again